### PR TITLE
feat(sound): add alt increment sound and fix ISI sound events.

### PR DESCRIPTION
**Pull request by ImenesFBW**
## Summary of Changes
- Adds sound to Altitude increment Selector (100/1000)

- Fixes the sound events for the ISI that went quiet after the react rework.

## Testing instructions
Make sure both the alt increment selector and ISIS buttons and knob produce sound.  
Fly a short flight and try to break the altitude increment selector by switching it a lot and changing the altitude with both increments to check if any new issues are introduced. 

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded. 

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Review by hotshotp — APPROVED** [review]
LGTM 😊

File: src/behavior/src/A32NX_Interior_Autopilot.xml [hunk 1]
```diff
@@ -213,4 +213,54 @@
...
+				<SET_STATE_ON>1000 (&gt;L:XMLVAR_Autopilot_Altitude_Increment)</SET_STATE_ON>
+				<STR_STATE_OFF>Hundred</STR_STATE_OFF>
+				<STR_STATE_ON>Thousand</STR_STATE_ON>
+				<TT_VALUE_OFF>'100'</TT_VALUE_OFF>
+				<TT_VALUE_ON>'1000'</TT_VALUE_ON>
+				<CHECK_STATE_HAS_CHANGED>False</CHECK_STATE_HAS_CHANGED>
+				<LOCAL_VAR_TO_WATCH_0>XMLVAR_Autopilot_Altitude_Increment</LOCAL_VAR_TO_WATCH_0>
+                <WWISE_EVENT_1>lswitch</WWISE_EVENT_1>
+			    <WWISE_EVENT_2>lswitch</WWISE_EVENT_2>
+            </UseTemplate>
```
Comment (tracernz) [inline src/behavior/src/A32NX_Interior_Autopilot.xml:252]: Mix of tabs and spaces all through here, please use spaces.

**Review by tracernz — CHANGES_REQUESTED** [review]

File: src/behavior/src/A32NX_Interior_Autopilot.xml (hunk 1 above)
Comment (ImenesFBW) [reply src/behavior/src/A32NX_Interior_Autopilot.xml]: Nice catch. Sorry I imported the asobo template and didn't spot that was going on. Should hopefully be all spaces now.

**Comment by marcman86** [comment]
Quality Assurance Tester Report
Discord:marcman86#4907
Object of testing: #6156
Tier of Testing: 1
Date: 14/11/2021

Testing Process:
- Spawned C&D at EDDP
- Tested altitude selector sound
- Tested ISIS button sounds
- Tested some other buttons
- Spawned in air
- Changed flight altitude in both ranges
- Fast switching and checked if it still works

Negatives: - 

Testing Results: Passed
Conclusions: Works for me

Hope this helps :-)

**Review by Benjozork — APPROVED** [review]

**Pull request merged**